src/uu/seq/src/seq.rs

```diff
@@ -240,6 +254,122 @@ pub fn uu_app() -> Command {
         )
 }
 
+// Fast code path increment function.
+// Add inc to the string val[start..end]. This operates on ASCII digits, assuming
+// val and inc are well formed.
+// Returns the new value for start.
```

Comment:
You can make this a docstring, which makes it also show up in language servers and things like that.
```suggestion
/// Fast code path increment function.
///
/// Add inc to the string val[start..end]. This operates on ASCII digits, assuming
/// val and inc are well formed.
/// Returns the new value for start.
```